sermon-mount.txt

```diff
@@ -1,217 +1,213 @@
+6 And blessed are all they who do hunger and thirst after righteousness, for they shall be filled with the Holy Ghost.
 
-7 Blessed are the merciful: for they shall obtain mercy.
+7 And blessed are the merciful, for they shall obtain mercy.
 
-8 Blessed are the pure in heart: for they shall see God.
+8 And blessed are all the pure in heart, for they shall see God.
 
-9 Blessed are the peacemakers: for they shall be called the children of God.
+9 And blessed are all the peacemakers, for they shall be called the children of God.
 
-10 Blessed are they which are persecuted for righteousness’ sake: for theirs is the kingdom of heaven.
+10 And blessed are all they who are persecuted for my name’s sake, for theirs is the kingdom of heaven.
 
-11 Blessed are ye, when men shall revile you, and persecute you, and shall say all manner of evil against you falsely, for my sake.
+11 And blessed are ye when men shall revile you and persecute, and shall say all manner of evil against you falsely, for my sake;
 
-12 Rejoice, and be exceeding glad: for great is your reward in heaven: for so persecuted they the prophets which were before you.
+12 For ye shall have great joy and be exceedingly glad, for great shall be your reward in heaven; for so persecuted they the prophets who were before you.
 
-13 ¶ Ye are the salt of the earth: but if the salt have lost his savour, wherewith shall it be salted? it is thenceforth good for nothing, but to be cast out, and to be trodden under foot of men.
+13 Verily, verily, I say unto you, I give unto you to be the salt of the earth; but if the salt shall lose its savor wherewith shall the earth be salted? The salt shall be thenceforth good for nothing, but to be cast out and to be trodden under foot of men.
 
-14 Ye are the light of the world. A city that is set on an hill cannot be hid.
+14 Verily, verily, I say unto you, I give unto you to be the light of this people. A city that is set on a hill cannot be hid.
 
-15 Neither do men light a candle, and put it under a bushel, but on a candlestick; and it giveth light unto all that are in the house.
+15 Behold, do men light a candle and put it under a bushel? Nay, but on a candlestick, and it giveth light to all that are in the house;
 
-16 Let your light so shine before men, that they may see your good works, and glorify your Father which is in heaven.
+16 Therefore let your light so shine before this people, that they may see your good works and glorify your Father who is in heaven.
 
-17 ¶ Think not that I am come to destroy the law, or the prophets: I am not come to destroy, but to fulfil.
+17 Think not that I am come to destroy the law or the prophets. I am not come to destroy but to fulfil;
 
-18 For verily I say unto you, Till heaven and earth pass, one jot or one tittle shall in no wise pass from the law, till all be fulfilled.
+18 For verily I say unto you, one jot nor one tittle hath not passed away from the law, but in me it hath all been fulfilled.
```

Comment:
the law had already been fulfilled by this point, since Christ had already suffered, died and been resurrected.